subsys/bluetooth/mesh/shell_config_cli.c

```diff
@@ -0,0 +1,1713 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/bluetooth/mesh.h>
+
+#include "net.h"
+#include "access.h"
+#include "shell_utils.h"
+
+#define CID_NVAL 0xffff
+
+static struct {
+	uint16_t dst;
+	uint16_t net_idx;
+} net = {
```

Comment:
Could you elaborate a bit?

Reply:
Previously, this structure was used not only by config client model, but also by other commands (e.g. `net-send`, health client, etc.). It would be good to preserve that behavior.

Reply:
Ok, I guess it could be preserved through utils?

Reply:
If it fits there, why not?